projects/storefrontlib/src/events/page/page.events.ts

```diff
@@ -14,4 +14,4 @@ export class PageVisitedEvent {
 /**
  * Indicates that a user visited the home page of a web presence.
```

Comment:
Can we leave "of a web presence". Sounds too academic and confuses.

Reply:
Done.